core/src/components/cat-input/cat-input.tsx

```diff
@@ -271,16 +280,6 @@ export class CatInput {
     this.catChange.emit(this.value);
   }
 
-  /**
-   * Adds a Cleave.js mask to the input.
-   *
-   * @param options The Cleave.js options.
-   */
-  @Method()
-  async mask(options: CleaveOptions): Promise<void> {
```

Comment:
Keep in mind that this is a breaking change. we need to build the version accordingly